playwright.config.ts

```diff
@@ -40,7 +40,7 @@ export default defineConfig<PluginOptions>({
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
   reporter: 'html',
   /* Retry on CI only */
-  retries: process.env.CI ? 2 : 0,
+  retries: process.env.CI ? 1 : 0,
```

Comment:
Increasing the amount of retries doesn't seem to help reduce flakey CI executions, but it does make the suite take a good bit longer to fail